ui/app/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -0,0 +1,325 @@
+        <div className="request-signature__header-background" />
+
+        <div className="request-signature__header__text">
+          { this.context.t('sigRequest') }
+        </div>
+
+        <div className="request-signature__header__tip-container">
+          <div className="request-signature__header__tip" />
+        </div>
+      </div>
+    )
+  }
+
+  renderAccount = () => {
+    const { selectedAccount } = this.state
+
+    return (
+      <div className="request-signature__account">
+        <div className="request-signature__account-text">
+          { this.context.t('account') + ':' }
+        </div>
+
+        <div className="request-signature__account-item">
+          <AccountListItem
+            account={selectedAccount}
+            displayBalance={false}
+          />
+        </div>
+      </div>
+    )
+  }
+
+  renderBalance = () => {
+    const { balance, conversionRate } = this.props
+
+    const balanceInEther = conversionUtil(balance, {
+      fromNumericBase: 'hex',
+      toNumericBase: 'dec',
+      fromDenomination: 'WEI',
+      numberOfDecimals: 6,
+      conversionRate,
+    })
+
+    return (
+      <div className="request-signature__balance">
+        <div className="request-signature__balance-text">
+          { `${this.context.t('balance')}:` }
+        </div>
+        <div className="request-signature__balance-value">
+          { `${balanceInEther} ETH` }
+        </div>
+      </div>
+    )
+  }
+
+  renderRequestIcon = () => {
+    const { requesterAddress } = this.props
+
+    return (
+      <div className="request-signature__request-icon">
+        <Identicon
+          diameter={40}
+          address={requesterAddress}
+        />
+      </div>
+    )
+  }
+
+  renderAccountInfo = () => {
+    return (
+      <div className="request-signature__account-info">
+        { this.renderAccount() }
+        { this.renderRequestIcon() }
+        { this.renderBalance() }
+      </div>
+    )
+  }
+
+  renderRequestInfo = () => {
+    return (
+      <div className="request-signature__request-info">
+        <div className="request-signature__headline">
+          { this.context.t('yourSigRequested') }
+        </div>
+      </div>
+    )
+  }
+
+  msgHexToText = (hex) => {
+    try {
+      const stripped = ethUtil.stripHexPrefix(hex)
+      const buff = Buffer.from(stripped, 'hex')
+      return buff.length === 32 ? hex : buff.toString('utf8')
+    } catch (e) {
+      return hex
+    }
+  }
+
+  renderTypedData = (data) => {
+    const { domain, message } = JSON.parse(data)
+    return (
+      <div className="request-signature__typed-container">
+        {
+          domain
+            ? <div>
+              <h1>
+                { 'Domain' }
+              </h1>
+              <ObjectInspector data={domain} expandLevel={1} name={'domain'} />
+            </div>
+            : ''
+        }
+        {
+          message
+            ? <div>
+              <h1>
+                {'Message'}
```

Comment:
This doesn't need to be a string expression in a JS, could we inline this into the "HTML" bit?